src/components/SignIn.js

```diff
@@ -16,6 +17,10 @@ const SignInPage = ({ history }) =>
     </div>
   );
 
+SignInPage.propTypes = {
+  history: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
```

Comment:
It's recommended to use `selleo-config` for eslint. Look into https://github.com/Selleo/eslint-config-react :)